govcd/lb_test.go

```diff
@@ -106,19 +108,46 @@ func (vcd *TestVCD) Test_LB(check *C) {
 	ruleDescription := addFirewallRule(vdc, vcd, check)
 	fmt.Printf("Done\n")
 
-	buildLoadBalancer(ip1, ip2, vcd, check)
+	// Build load balancer
+	buildLoadBalancer(edge, ip1, ip2, vcd, check)
+
+	// Cache current load balancer settings for change validation in the end
+	beforeLb, beforeLbXml := testCacheLoadBalancer(edge, check)
+
+	// Enable load balancer globally
```

Comment:
In the interest of failing quickly, can we move this operation to the top? That way, if this operation fails, we don't need to wait for the longer tasks to complete.

Reply:
I want to keep it there for a few reasons:
* If I enable load balancing before all the configuration is created (`buildLoadBalancer(edge, ip1, ip2, vcd, check)`) then there is less configuration available in load balancer and less chances to catch corrupt config (matching raw XMLs)
* `testCacheLoadBalancer` and `testCheckLoadBalancerConfig` `XML` validationg would not work in this case because majority of load balancer component cleanup is left for teardown part in the end of suite